app/assemblers/assemble_exercise_community_solutions_list.rb

```diff
@@ -22,7 +22,11 @@ def solutions
     Solution::SearchCommunitySolutions.(
       exercise,
       page: params[:page],
-      criteria: params[:criteria]
+      order: params[:order],
+      criteria: params[:criteria],
+      sync_status: params[:up_to_date] == "true" ? :up_to_date : nil,
```

Comment:
Is this how we check for booleans in other places? I'd have thought Rails would magically do this for us.